### DISCUSSION TITLE
modular code generation

### DISCUSSION THREAD

**Discussion by louiskhchan** [simolus3/drift#2339] [Q&A] [answered]
Hi there. 

The modular code generation builder option has been available since version 2.3.0, and the website indicates that there were expected breaking changes up to version 2.4.0. Could you advise me on the stability status of this feature in version 2.5.0, and comment on whether it is ready for production use?

Thank you very much for your help.

**Comment by simolus3** [comment, marked as answer]
The main reason I called it experimental was an uncertainty about potential design issues that would require larger, user-visible changes. But all of the recently fixed issues were in the implementation and didn't require a change in the user-visible API. I'm confident that the design for modular code generation is stable and ready for production use, I'll remove the warning from the documentation.

If any issues come up, I'll do my best to fix them quickly :)

**Reply by louiskhchan** [reply]
Thank you for your quick and helpful answer. I’m glad to use the modular code generation feature. Your software is awesome! 😊
